src/main/java/fi/hsl/transitdata/hfp/Deduplicator.java

```diff
@@ -76,7 +76,8 @@ private byte[] parsePayload(Message received) {
                 .filter(transitdataSchema ->
                         //We only support these two protobuf formats. Add others if needed:
                         transitdataSchema.schema.equals(TransitdataProperties.ProtobufSchema.MqttRawMessage) ||
-                        transitdataSchema.schema.equals(TransitdataProperties.ProtobufSchema.HfpData)
+                        transitdataSchema.schema.equals(TransitdataProperties.ProtobufSchema.HfpData) ||
+                        transitdataSchema.schema.equals(TransitdataProperties.ProtobufSchema.PassengerCount)
```

Comment:
These changes are from previous commit. They're tested in dev, but kinda misleading to merge them in PR 'Increase maximum cache size'.

Make sure it's ok to merge these too before proceeding.